### DISCUSSION TITLE
How to integrate seleniumbase in visual studio project

### DISCUSSION THREAD

**Discussion by iqbalshekh** [seleniumbase/SeleniumBase#755] [Q&A] [answered]
Hi,

I have ready project for automation build on python Unitest framework. After studying seleniumbase I wants to add seleniumbase framework in to my existing project. But I confusion about to create new project in VS or use existing one.

can you suggest better option?

Thanks,

**Comment by mdmintz** [comment, marked as answer]
Hi @iqbalshekh, as long as you use a separate Python virtual environment from the rest of your Python code, there should be no problem adding SeleniumBase into an existing project. The separate virtual environment will prevent dependency conflicts.
